src/approaches/interactive_learning_approach.py

```diff
@@ -92,27 +92,20 @@ def learn_from_offline_dataset(self, dataset: Dataset) -> None:
                 self._simulator,
                 self._get_current_predicates(),
                 max_steps=CFG.interactive_max_steps)
-            # Decide whether to ask at each state during exploration
+            # Decide whether to ask about each possible atom during exploration
             for s in traj.states:
-                abstract_state = utils.abstract(s,
-                                                self._get_current_predicates())
-                score = score_goal(self._dataset_with_atoms, abstract_state)
-                # Ask about this state if it is the best seen so far
-                if score > best_score:
-                    # Pick best ground atom in the state
-                    ground_atoms = utils.all_possible_ground_atoms(
-                        s, self._predicates_to_learn)
-                    atom_set = get_best_goal_by_score(self._dataset_with_atoms,
-                                                      [{ga}
-                                                       for ga in ground_atoms])
-                    assert len(atom_set) == 1
-                    atom = atom_set.pop()
-                    if self._ask_teacher(s, atom):
-                        # Add this atom if it's a positive example
-                        self._dataset_with_atoms.append(  # pragma: no cover
-                            (LowLevelTrajectory([s], []), [{atom}]))
-                        # Still need to implement a way to use negative examples
-                    best_score = score
+                ground_atoms = utils.all_possible_ground_atoms(
+                    s, self._predicates_to_learn)
+                for atom in ground_atoms:
+                    score = score_atom(self._dataset_with_atoms, atom)
```

Comment:
add a comment so we don't forget: although this score function currently does not use `s`, we imagine that future score functions importantly will.